opus/application/static_media/js/opus.js

```diff
@@ -37,22 +37,24 @@ var opus = {
     lastRequestNo: 0,          // holds request numbers for main result count loop,
     lastAllNormalizeRequestNo: 0,
     lastResultCountRequestNo: 0,
+    lastNormalizeurlRequestNo: 0,
     waitingForAllNormalizedAPI: false,
 
     // client side prefs, changes to these *do not trigger results to refresh*
     // prefs gets added verbatim to the url, so don't add anything weird into here!
-    prefs:{ 'view':'search', // search, browse, cart, detail
-            'browse':'gallery', //either 'gallery' or 'data'
-            'cart_browse':'gallery',  // which view is showing on the cart page, gallery or data
-            'page':default_pages,  // what page are we on, per view, default defined in header.html
-                                   // like {"gallery":1, "data":1, "cart_gallery":1, "cart_data":1 };
-            'limit': 100, // results per page
-            'order': default_sort_order.split(','),  // result table ordering
-            'cols': default_columns.split(','),  // default result table columns by slug
-            'widgets':[], // search tab widget columns
-            'detail':'', // opus_id of detail page content
-
-
+    // prefs key:value pair order has been re-organized to match up with normalized url
+    prefs: {
+        "cols": default_columns.split(","),  // default result table columns by slug
+        "widgets": [], // search tab widget columns
+        "order": default_sort_order.split(","),  // result table ordering
+        "view": "search", // search, browse, cart, detail
+        "browse": "gallery", //either 'gallery' or 'data'
+        "startobs": 1,
```

Comment:
Debby: Is there now going to be a cart_startobs, so that the main gallery and the cart page can have different scroll positions saved? If so we need to add it to a bunch of places in the Django.

Reply:
Ok I will add it to the back end after this pull is finished. I have variety of other changes to make to normalize url as well.